projects.css

```diff
@@ -0,0 +1,251 @@
+/* Add your CSS here */
+
+/* Dont' forget to link this file to your HTML in the <head> */
+
+/* header */
+header {
+    position: sticky;
+    background-color: white;
+}
+.header-wrapper{
+    display: flex;
+    justify-content: space-between;
+    max-width: 1400px;
+    margin: 0 auto;
+}
+
+.header-menu{
+    display: flex;
+    
+}
+
+.header-menu li {
+    padding: 20px;
+    list-style: none;
+    font-size: 0.9rem;
+    font-family: 'Quicksand', sans-serif;
+    
+}
+
+.header-navigation a {
+    text-decoration: none;
+    color: black;
+}
+
+.header-name{
+    font-size: 1.6rem;
+    font-family: 'Poppins', sans-serif;
+}
+
+.project-manager{
+    font-size: 1rem;
+    font-family: 'Quicksand', sans-serif;
+}
+header {
+    top: 0;
+    left: 0;
+    width: 100%; 
+}
+
+.square {
+    background-color: blue;
+    height: 15px;
+    width: 15px;
+}
+
+.square-name {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+/* main */
+
+.content-wrapper{
+    display: flex;
+    flex-direction: column;
+    height: 90vh;
+    gap: 2rem;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.background-dark-beige {
```

Comment:
Have a look into CSS linear-gradient. Here's a codepen you might find interesting.  https://codepen.io/stefanjudis/pen/vYOjJdZ?editors=1100